docs/site/schema-design.md

```diff
@@ -34,11 +43,11 @@ erDiagram
     Patient {
         bigint id PK "Primary Key (auto-generated)"
         bigint person_id FK "Foreign Key to Person"
-        json data "Patient Data JSON Object"
+        json data "Patient Data"
         uuid reference_id "Reference UUID (auto-generated)"
         string external_patient_id "External Patient ID"
         string external_person_id "External Person ID"
-        string external_person_source "External Person Source"
+        string external_person_source "Source System"
```

Comment:
Should we keep this as External Source System just so it matches "External Patient ID" and "External Person ID"? I can see it going both ways, where 'external' might be an odd way to refer to a surveillance system (for example). I don't have particularly strong feelings on this one, just wanted to double check.